README.md

```diff
@@ -1,3 +1,42 @@
-# learning
+# Learning
 It is my first repository, there will be all information about my learning
 
+## Table of contents
+* [Links](#links)
+* [Git Commands](#git-commands)
+
+## Links
+Readme:
+[python](https://www.python.org/downloads/)
```

Comment:
LINKS TO README not python, pycharm, git, notepad

Reply:
Done